mappings/net/minecraft/block/entity/BannerPattern.mapping

```diff
@@ -1,54 +1,14 @@
 CLASS net/minecraft/class_2582 net/minecraft/block/entity/BannerPattern
-	CLASS class_3750 Builder
+	CLASS class_3750 ToTagger
```

Comment:
What...? This is a builder class. `ToTagger` would be a method name, and what's a tagger?

Reply:
This is not a builder. This is something that builds a tag, named like guava's `ToStringer`

Reply:
Interesting. I'd prefer something like `PatternTagBuilder` or `PatternListSerializer`, but I guess `ToTagger` works, if guava's using it too.